package.json

```diff
@@ -34,6 +34,7 @@
     "@angular/platform-browser": "2.3.0",
     "babel-cli": "^6.18.0",
     "babel-plugin-transform-react-jsx": "^6.8.0",
+    "babel-preset-es2015-rollup": "^3.0.0",
```

Comment:
Works for me, yep. When you say 'option', are you referring just to `modules` or also to `external-helpers`? (To clarify the reasoning behind `external-helpers`, since it's far from obvious — it allows us to create a synthetic module such that `babelHelpers.classCallCheck` etc become mangleable functions rather than properties of an object)